synapse/axon.py

```diff
@@ -308,6 +387,7 @@
         self.onfini(self.blobslab.fini)
 
     def _initAxonHttpApi(self):
+        self.addHttpApi('/api/v1/axon/files/del', AxonHttpDelV1, {'cell': self})
         self.addHttpApi('/api/v1/axon/files/put', AxonHttpUploadV1, {'cell': self})
         self.addHttpApi('/api/v1/axon/files/has/sha256/([0-9a-fA-F]{64}$)', AxonHttpHasV1, {'cell': self})
         self.addHttpApi('/api/v1/axon/files/by/sha256/([0-9a-fA-F]{64}$)', AxonHttpDownloadV1, {'cell': self})
```

Comment:
Given the new method, maybe we should rename this to something like AxonHttpByV1?